#Expyriment 0.9.0 (Python 3.6.6), .xpe-file, coding: ISO8859-1
#date: Sat Aug 25 2018 15:21:38
#sha1: None
#modules: 
#display: size=(800, 600), window_mode=True, open_gl=2
#os: uname_result(system='Linux', node='L3460', release='4.15.0-33-generic', version='#36-Ubuntu SMP Wed Aug 15 16:00:05 UTC 2018', machine='x86_64', processor='x86_64')
Time,Type,Event,Value,Detail,Detail2
300,stdout,received,"[['yellow', (255, 255, 0)], ['yellow', (255, 255, 0)], ['green', (0, 255, 0)], ['red', (255, 0, 0)], ['red', (255, 0, 0)], ['blue', (0, 0, 255)], ['red', (255, 0, 0)], ['yellow', (255, 255, 0)], ['red', (255, 0, 0)], ['blue', (0, 0, 255)], ['red', (255, 0, 0)], ['yellow', (255, 255, 0)], ['blue', (0, 0, 255)], ['red', (255, 0, 0)], ['blue', (0, 0, 255)], ['blue', (0, 0, 255)], ['yellow', (255, 255, 0)], ['yellow', (255, 255, 0)], ['red', (255, 0, 0)], ['yellow', (255, 255, 0)], ['red', (255, 0, 0)], ['blue', (0, 0, 255)], ['blue', (0, 0, 255)], ['yellow', (255, 255, 0)], ['green', (0, 255, 0)], ['yellow', (255, 255, 0)], ['blue', (0, 0, 255)], ['blue', (0, 0, 255)]]"
300,stdout,received,'10'
300,stdout,received,'16'
301,stdout,received,'28'
301,stderr,received,'Traceback (most recent call last):'
301,stderr,received,'  File "run_nback.py", line 26, in <module>'
301,stderr,received,'    b=a(COLORS)                #fn call'
301,stderr,received,"NameError: name 'a' is not defined"
